navigation/tracking/scripts/pdaf.py

```diff
@@ -0,0 +1,250 @@
+import numpy as np
+
+"""
+Single object tracking
+
+Sub tasks: 
+
+    Use a kalam gain so that the filter is numerically stable. 
+    SINGULAR MATRICIES
+
+    TEST
+        - Visualize position estimates with clutter.
+        - Visualize velocity estimates with vectors. 
+        - Visualize validation gate. 
+        - Does the frequency of "zero detections within gate" correspond with p of detection? 
+
+    Observations: 
+        P pri and post are not symmetrical.
+        P post grows to values over 1000.
+"""
```

Comment:
I think this docstring is a good place to add references, since this implementation is somewhat theoretically heavy for readers who have not had TTK4250 😛 

Some comments reference Brekke, but I couldn't find anything referring to the slides you showed at some point. Might be a good idea to cite those slides in a similar manner, and drop the link to the slides up here